### DISCUSSION TITLE
No "integrated" assay after running CCA integration on an SCTransformed object

### DISCUSSION THREAD

**Discussion by NicoHadas** [satijalab/seurat#8810] [Q&A]
Hello,

I have ran both SCTransform and CCA Integration on my objec but cannot find an "integrated" assay.

I have followed the [Perform integration with SCTransform-normalized datasets](https://satijalab.org/seurat/articles/integration_introduction#perform-integration-with-sctransform-normalized-datasets) exactly.

Please advise.

**Comment by SMP7UC** [comment]
Hi, 

I am also following the same link to perform CCA integration, and I am also not finding the "integrated" assay as earlier. 
I am also confused about re-clustering the subset of cells taken from a seurat object. The following is the code I use for re-clustering. Any suggestions in this regard will be very helpful. 

neu_rcl = Fil_neu %>%
 RunPCA() %>%
  FindNeighbors(reduction = "pca", dims = 1:30) %>%
  FindClusters() %>%
  RunUMAP(dims = 1:30)
